src/components/CTABlock.tsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from 'react'
+import { EventControlledContent } from './EventControlledContent'
+
+interface CTABlockProps {
+  contentString: string
+  contentPrompt: string
+  buttonText: string
+  fireEvent?: string
+  waitForEvent?: string
+
+}
+
+export const CTABlock = ({ contentString, contentPrompt, buttonText, fireEvent, waitForEvent }: CTABlockProps): JSX.Element => {
+  const [clicked, setclicked] = useState<boolean>(false)
+
+  const clickHandler = (): void => {
+    if (fireEvent !== undefined) {
+      const clickEvent = new CustomEvent(fireEvent)
+      document.dispatchEvent(clickEvent)
+    }
+    setclicked(true)
+  }
+
+  const promptButton = (): JSX.Element => {
```

Comment:
Minor: Can we rename this to something like `maybePromptAndButton`? I otherwise get confused when reading the render block :sweat_smile:.